registry/test/utils/postgres.js

```diff
@@ -7,7 +7,12 @@ const { Client } = require('pg');
 
 function providePostgres(to) {
   return async function(...args) {
-    const client = new Client();
+    const client = new Client({
+      user: process.env.PGUSER,
+      database: process.env.PGDATABASE,
+      port: 5432,
+      host: process.env.POSTGRES_HOST
```

Comment:
A-ha! So! node-pg (and many other postgres tools) will use `PGUSER` / `PGDATABASE` automatically if they're set in the environment. If we switch from `POSTGRES_HOST` to `PGHOST` the original code _should_ work.

Reply:
Oh good to know! It looks like the env var CJ added in Circle fixed it, but I changed the names anyway in case you'd rather have that in code.